server/app/views/BaseHtmlLayout.java

```diff
@@ -158,4 +181,162 @@ private ImmutableList<ScriptTag> getAnalyticsScripts(String trackingTag) {
             .withType("text/javascript");
     return new ImmutableList.Builder<ScriptTag>().add(scriptImport).add(rawScript).build();
   }
+
+  /**
+   * Creates the banner which indicates that CiviForm is an official government website. This banner
+   * is at the top of every page. It is a USWDS component:
+   * https://designsystem.digital.gov/components/banner/
+   *
+   * @return a html section tag
+   */
+  protected SectionTag getGovBanner(Optional<Messages> maybeMessages) {
```

Comment:
I chose to put this method in `BaseHtmlLayout` so that both `ApplicantLayout` and `AdminLayout` could make use of it.  That's also why the `maybeMessages` parameter is optional.  The ApplicantLayout needs the translated messages, but the AdminLayout doesn't.